core/themes/bartik/templates/node.tpl.php

```diff
@@ -77,9 +77,8 @@
  * @see template_preprocess_node()
  */
 ?>
-<article id="node-<?php print $node->nid; ?>" class="<?php print implode(' ', $classes); ?> clearfix"<?php print backdrop_attributes($attributes); ?>>
+<article id="node-<?php print $node->nid; ?>" class="<?php print implode(' ', $classes); ?> clearfix"<?php print backdrop_attributes($attributes); ?> role="article">
```

Comment:
Perhaps, I don't actually recall, embarrassingly. If it shouldn't be so, we can kill it.